controllers/articles.controllers.js

```diff
@@ -23,12 +22,31 @@ exports.getArticles = (req, res, next) => {
   });
 };
 
-exports.getComments = (req, res, next) => {
+exports.patchArticle = (req, res, next) => {
   const { article_id } = req.params;
+  const { inc_votes } = req.body;
 
-  Promise.all([fetchComments(article_id), checkArticleExists(article_id)])
-    .then(([comments]) => {
-      res.status(200).send({ comments });
+  const inputBodyFieldsArray = Object.keys(req.body);
+  const fieldSchema = {
+    inc_votes: "number",
+  };
+
+  if (inputBodyFieldsArray.length !== 1) {
+    return res.status(400).send({ status: 400, message: "invalid body" });
+  }
+
+  inputBodyFieldsArray.forEach((field) => {
+    if (typeof req.body[field] !== fieldSchema[field]) {
+      return res.status(400).send({ status: 400, message: "invalid body" });
+    }
+  });
```

Comment:
Please check for the right patch object in the model and deal with responses in your error handling middleware (app.js).